### ISSUE TITLE
Open Beelden collection B&G is not working properly (search does not work)

### ISSUE THREAD

**Issue by lilimelgar** [CLARIAH/wp5_mediasuite#72]
Tested in version 2 (released Dec.2017), search does not work in Open Beelden B&G.

**Comment by lilimelgar** [comment]
Willem says: 
Open Beelden doesn't seem to be indexed. I will have a look at it. For now, I'll remove the elasticsearch/json format in CKAN.
Also, all polygoon is available within the Collectie Beeld en Geluid.

**Comment by lilimelgar** [comment]
I added a "priority" label to this, since it could be an interesting case to explain the differences between the Comparative search recipe and the exploratory search recipe (DIVE), since this is one of the few (if not the only one) collection they have in common... Of course, if not possible this time, we leave it for V2.

**Comment by wmelder** [comment]
Look at it, but there seems to be an issue with the OAI harvester. I thought I fixed it, but still not indexing yet..

**Comment by wmelder** [comment]
I figured out that the Open Beelden server doesn't comply with the OAI-PMH protocol on handling HTTP POST requests.
https://www.openarchives.org/OAI/openarchivesprotocol.html#HTTPRequestFormat
Therefor, the collection isn't harvested in the MediaSuite. We need to discuss whether we should provide in a fix in the importer or forward the issue to Open Beelden.

**Comment by wmelder** [comment]
Also, the mediasuite data should be refreshed, because the Open Beelden collection is provided in the collection selector, while it is not tagged as Clariah Media Suite indexed in CKAN.

**Comment by jblom** [comment]
still need to implement a daily or hourly refresh of the CKAN collection data in the MS. Now it only refreshes on restart and only after you delete the cached file.

**Comment by lilimelgar** [comment]
The Open Beelden collections are available in DIVE. If they become available in the other recipes of the Media Suite we could design a comparative user study between two recipes with the same collection. This could be an interesting study for DIVE, but it is not clear yet how important the Open Beelden collections are for the NARDIS project. Liliana will ask around and indicate the priority of this issue after May 17, it is archived for the moment.

**Comment by lilimelgar** [comment]
@wmelder You mentioned there was a solution for this issue? If so, shall we close it?

**Comment by wmelder** [comment]
Now there's another issue I saw yesterday. Need fixing

Verstuurd vanaf mobiel (
dus kort)

**Comment by lilimelgar** [comment]
Just to check: is this an "easy fix" or too complex/time consuming? To know if we can use this collection for testing annotation in early July.

**Comment by jblom** [comment]
I misconfigured the search API for the media suite causing it to restrict access to the B&G openbeelden index. I corrected this and now you can search Open Beelden in the media suite.

So please test this s'il vous plait

**Comment by lilimelgar** [comment]
Tested in version 2 (production version), inspector, search, comparative. Working well now!
